frontend/src/Components/Pages/SignUpPage.js

```diff
@@ -1,10 +1,47 @@
 import { Button, Card, Container, Form } from 'react-bootstrap';
 import { useNavigate } from "react-router-dom";
+import { useState } from "react";
 
 function SignUpPage() {
 
     let navigate = useNavigate();
 
+    const [full_name, setName] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [accountType, setAccountType] = useState("Customer");
+
+    const handleSubmit = async (event) => {
+        event.preventDefault();
+
+        try {
+            let res = await fetch("http://localhost:3001/storeUserAccountInfo", {
+            method: "POST",
+            mode: "cors",
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify({
+                full_name: full_name,
+                email: email,
+                password: password,
+                account_type: accountType,
+                }),
+            });
+            let resJson = await res.json();
+            if (res.status === 200) {
+                setName("");
+                setEmail("");
+                setPassword("");
+                setAccountType("Customer");
+
+                // TODO: User created successfully, redirect to correct page here
```

Comment:
To redirect to home page you can use the same function as on line 48: 

`navigate('/')` 